javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -244,8 +244,11 @@ abstract class Configuration extends string {
    * EXPERIMENTAL. This API may change in the future.
    *
    * Holds if `pred` should be stored in the object `succ` under the property `prop`.
+   * The object `succ` must be a `DataFlow::SourceNode` for the object wherein the value is stored.
    */
-  predicate isAdditionalStoreStep(DataFlow::Node pred, DataFlow::Node succ, string prop) { none() }
+  predicate isAdditionalStoreStep(DataFlow::Node pred, DataFlow::SourceNode succ, string prop) {
```

Comment:
@asgerf FYI: this is technically a breaking change to an `EXPERIMENTAL` API, so if your WIP branches start breaking, then this may be the cause.